test/expressions/generateExpressionTests.py

```diff
@@ -44,7 +44,7 @@ def make_arg_code(arg, scalar, var_name, var_number, function_name):
         return_type = arg_types[arg.split(" => ")[1]].replace("SCALAR", scalar)
         return "  stan::test::test_functor " + var_name
     arg_type = arg_types[arg].replace("SCALAR", scalar)
-    if function_name in special_arg_values:
+    if function_name in special_arg_values and arg != "rng":
```

Comment:
```suggestion
    if (function_name in special_arg_values) and (arg != "rng"):
```

That is fine, I usually add parenthesis just to increase some readability.